source/jobs/prerequisites/oracle.txt

```diff
@@ -28,13 +28,18 @@ About this Task
 
 - If you're migrating from an Oracle 12c instance, you must run commands as 
   the SYSDBA role.
-- In Oracle 12c the concept of a pluggable database was introduced. Some
-  of these commands can be run on PDB(plugable database) while commands 
+- Oracle 12c introduced the concept of a pluggable database. Some
+  of these commands can be run on a PDB (pluggable database), while commands 
   such as enabling ``ARCHIVELOG`` must be run on the
-  CDB(container/master database). For details on each architecture, 
+  CDB (container/master database). For details on each architecture, 
   see `Overview of Container Databases and Pluggable Databases
   <https://oracle-base.com/articles/12c/multitenant-overview-container-database-cdb-12cr1>`__.
-- Supplemental logging is not allowed in Oracle Express editions.
+- Some commands differ based on whether the database is single or
+  multi-tenant. In a multi-tenant database, permissions must
+  include the suffix ``CONTAINER=ALL``
+- Oracle Express editions don't support logging the necessary information for
```

Comment:
```suggestion
- Oracle Database Express Edition (XE) doesn't support logging the necessary information for
```
But perhaps, 
"You can't run a continuous migration against an Oracle Database Express Edition (XE) because XE doesn't support the necessary logs for continuous migration jobs."